generator/generator-opc-generator/generators/app/templates/src/_style.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../opc-styles/opc-styles.scss";
+
+$LOCAL: <%= componentName %>;
```

Comment:
@hybridx u need to create a lowercase component name, in scss we won't be relying on the case that user has provided.

Reply:
Thanks we should change the case to lowercase by default and remove spaces for `-` hyphens